### PR TITLE
[0.10.5] Schedule Editor for Admins

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
## Features
- Redesigned Admin Panel
- Schedule Editor for admins
  Resolves FRO-146 [(link)](https://linear.app/skiso/issue/FRO-146/simple-schedule-editor-for-admins)

## Code Changes
- Modified Schedule to be more flexible

## Fixes
- Non-breaking space before mai yamok

## Preview
Preview available at https://pr1.mysk.school/.

### CODE REVIEW

File: public/static/locales/th/manage.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
   },
   "classrooms": {
     "title": "ห้องเรียน",
-    "desc": "นักเรียนได้รับการจัดกลุ่มเป็นห้องเรียนต่าง ๆ ทุกปี พิมพ์ใบรายชื่อของทุกห้องเรียนพร้อมกันได้ที่นี่",
+    "desc": "นักเรียนได้รับการจัดกลุ่มเป็นห้องเรียนต่าง ๆ ทุกปี พิมพ์ใบรายชื่อของทุกห้องเรียนพร้อมกันได้ที่นี่",
```
Comment (pixelpxed) [inline public/static/locales/th/manage.json:51]: How is this different!? lol

**Review by pixelpxed — APPROVED** [review]
LTGM! I like the execution and the UI, looks cool and easy to use.

How does the edit for teacher work tho?
| pr1 mysk school_admin_schedule (iPad Pro) | pr1 mysk school_admin_schedule(iPad Pro) (1) |
|---|---|

File: public/static/locales/th/manage.json (hunk 1 above)
Comment (SiravitPhokeed) [reply public/static/locales/th/manage.json:51]: Mai yamok must be proceeded by a non-breaking space.

**Comment by SiravitPhokeed** [comment]
> How does the edit for teacher work tho?

@pixelpxed The whole point of this UI is to repurpose Schedule editing tools meant for Teachers to edit Classroom Schedules instead. Teachers already have their own Schedule editing pages.

**Pull request merged**